ast/enum.go

```diff
@@ -54,18 +54,15 @@ func NewEnumNode(keyword *KeywordNode, name *IdentNode, openBrace *RuneNode, dec
 	}
 	children := make([]Node, 0, 4+len(decls))
 	children = append(children, keyword, name, openBrace)
-	for _, decl := range decls {
```

Comment:
Felt like a good optimization to only iterate over this slice once (to assert the types and append to children).